<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlechuhrWindow</class>
 <widget class="QWidget" name="BlechuhrWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="userList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tab">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <widget class="QWidget" name="accountsTab">
         <attribute name="icon">
          <iconset resource="../res/blechuhr.qrc">
           <normaloff>:/blechuhr/images/blechuhr/accounts.svg</normaloff>:/blechuhr/images/blechuhr/accounts.svg</iconset>
         </attribute>
         <attribute name="title">
          <string>Accounts</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <layout class="QGridLayout" name="accountsLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="rulesTab">
         <attribute name="icon">
          <iconset resource="../res/blechuhr.qrc">
           <normaloff>:/blechuhr/images/blechuhr/rules.svg</normaloff>:/blechuhr/images/blechuhr/rules.svg</iconset>
         </attribute>
         <attribute name="title">
          <string>Rules</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <layout class="QGridLayout" name="rulesLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="dayTab">
         <attribute name="icon">
          <iconset resource="../res/blechuhr.qrc">
           <normaloff>:/blechuhr/images/blechuhr/day.svg</normaloff>:/blechuhr/images/blechuhr/day.svg</iconset>
         </attribute>
         <attribute name="title">
          <string>Day</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QCalendarWidget" name="dayCalendarWidget">
            <property name="firstDayOfWeek">
             <enum>Qt::Monday</enum>
            </property>
            <property name="gridVisible">
             <bool>false</bool>
            </property>
            <property name="navigationBarVisible">
             <bool>true</bool>
            </property>
            <property name="dateEditEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="3">
           <widget class="QWebView" name="dayWebView">
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Summary</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="1" column="2">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="dayDataDeleteButton">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="dayDataAddButton">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QPushButton" name="dayDataSaveButton">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="4">
              <widget class="QListWidget" name="dayListWidget"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="weekTab">
         <attribute name="icon">
          <iconset resource="../res/blechuhr.qrc">
           <normaloff>:/blechuhr/images/blechuhr/week.svg</normaloff>:/blechuhr/images/blechuhr/week.svg</iconset>
         </attribute>
         <attribute name="title">
          <string>Week</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QCalendarWidget" name="weekCalendarWidget"/>
          </item>
          <item row="0" column="1">
           <widget class="QWebView" name="weekWebView">
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="monthTab">
         <attribute name="icon">
          <iconset resource="../res/blechuhr.qrc">
           <normaloff>:/blechuhr/images/blechuhr/month.svg</normaloff>:/blechuhr/images/blechuhr/month.svg</iconset>
         </attribute>
         <attribute name="title">
          <string>Month</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QCalendarWidget" name="monthCalendarWidget"/>
          </item>
          <item row="2" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="4">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2" rowspan="2" colspan="3">
           <widget class="QWebView" name="monthWebView">
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Summary</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Hours</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="monthSummarylineEdit">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="summaryTab">
         <attribute name="icon">
          <iconset resource="../res/blechuhr.qrc">
           <normaloff>:/blechuhr/images/blechuhr/summary.svg</normaloff>:/blechuhr/images/blechuhr/summary.svg</iconset>
         </attribute>
         <attribute name="title">
          <string>Summary</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="1" column="1">
           <widget class="QPushButton" name="importButton">
            <property name="text">
             <string>Import CSV</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="exportButton">
            <property name="text">
             <string>Export CSV</string>
            </property>
            <property name="icon">
             <iconset resource="../res/blechuhr.qrc">
              <normaloff>:/blechuhr/images/blechuhr/csv_file.svg</normaloff>:/blechuhr/images/blechuhr/csv_file.svg</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="printToPDFButton">
            <property name="text">
             <string>Print to PDF</string>
            </property>
            <property name="icon">
             <iconset resource="../res/blechuhr.qrc">
              <normaloff>:/blechuhr/images/blechuhr/pdf_file.svg</normaloff>:/blechuhr/images/blechuhr/pdf_file.svg</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="importBlechuhrButton">
            <property name="text">
             <string>Import RFID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>580</width>
               <height>449</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QGridLayout" name="summaryGridLayout"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="exportTab">
         <attribute name="icon">
          <iconset resource="../res/blechuhr.qrc">
           <normaloff>:/blechuhr/images/blechuhr/export.svg</normaloff>:/blechuhr/images/blechuhr/export.svg</iconset>
         </attribute>
         <attribute name="title">
          <string>Export</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <layout class="QGridLayout" name="exportLayout"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/blechuhr.qrc"/>
 </resources>
 <connections/>
</ui>
